api/_mixins/LinkStyle__HTMLLinkElement.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "HTMLLinkElement": {
+      "sheet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLLinkElement/sheet",
+          "spec_url": "https://drafts.csswg.org/cssom/#the-linkstyle-interface",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "15"
```

Comment:
I'm surprised by this, but can confirm http://mdn-bcd-collector.appspot.com/tests/api/HTMLLinkElement/sheet shows no support in Opera 12.16.